dataactcore/scripts/pullFPDSData.py

```diff
@@ -0,0 +1,1124 @@
+import logging
+import argparse
+import requests
+import xmltodict
+
+import datetime
+import re
+
+from dataactcore.logging import configure_logging
+
+from sqlalchemy.dialects.postgresql import insert
+
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.models.domainModels import CGAC, SubTierAgency
+from dataactcore.models.stagingModels import DetachedAwardProcurement
+from dataactcore.models.jobModels import FPDSUpdate
+
+from dataactcore.models.jobModels import Submission  # noqa
+from dataactcore.models.userModel import User  # noqa
+
+from dataactvalidator.health_check import create_app
+
+feed_url = "https://www.fpds.gov/ezsearch/FEEDS/ATOM?FEEDNAME=PUBLIC&templateName=1.4.5&q="
+delete_url = "https://www.fpds.gov/ezsearch/FEEDS/ATOM?FEEDNAME=DELETED&templateName=1.4.5&q="
+
+logger = logging.getLogger(__name__)
+
+
+def list_data(data):
+    if isinstance(data, dict):
+        # make a list so it's consistent
+        data = [data, ]
+    return data
+
+
+def extract_text(data_val):
+    if type(data_val) is not str:
+        data_val = data_val['#text']
+    return data_val
+
+
+# Get values from the awardID level of the xml
```

Comment:
We should convert these kinds of comments that describe what the function is doing into docstings